File: Chapter 05/C#/Section_5_4_Server_Monitor/Before/AttemptedTest.cs
using System;

namespace Chapter05.ServerMonitor.Before
{
    public static class AttemptedTest
    {
        public static void Run()
        {
            Console.WriteLine("\n--- ATTEMPTING TO TEST (BEFORE) ---");
            
            var monitor = new ServerMonitor();

            // ACT
            Console.WriteLine("Test Action: Calling CheckTemperature(95)...");
            monitor.CheckTemperature(95);

            // ASSERT
            // ... Wait. How do we check if it worked?
            // We can't check 'monitor.SentMessages' because it doesn't exist.
            // We can't mock Twilio because it's 'new'd up' inside the class.
            
            Console.WriteLine("FAIL: Impossible to verify outcome programmatically.");
            Console.WriteLine("      (You have to manually check the console logs.)");
        }
    }
}

File: Chapter 05/Python/section_5_4_server_monitor/before/attempted_test.py
from .server_monitor import ServerMonitor

class AttemptedTest:
    @staticmethod
    def run() -> None:
        print("\n--- ATTEMPTING TO TEST (BEFORE) ---")
        
        monitor = ServerMonitor()

        # ACT
        print("Test Action: Calling check_temperature(96)...")
        monitor.check_temperature(96)

        # ASSERT
        # ... Wait. How do we check if it worked?
        # We can't check 'monitor.sent_messages' because it doesn't exist.
        # We can't mock Twilio because it's 'new'd up' inside the class.
        
        print("FAIL: Impossible to verify outcome programmatically.")
        print("      (You have to manually check the console logs.)")